modules/react/combobox/lib/ComboboxInput.tsx

```diff
@@ -40,21 +60,20 @@ export const useComboboxInput = composeHooks(
               `[id="${listboxId}"] [data-id="${model.state.cursorId}"]`
             );
             if (menuItem) {
-              menuItem.scrollIntoView({block: 'nearest'});
+              requestAnimationFrame(() => {
+                menuItem.scrollIntoView({block: 'nearest'});
```

Comment:
This prevents the popup from scrolling to the top of the page, re force another re render so that the browser knows where the popup is localted